scripts/save_avro.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python
+"""
+Script to save data from JSON files to an Avro file
+
+Usage:
+    $ python save_avro.py ingestion_document_id codec output_directory
+
+To use snappy support you need to install snappy with your package manager,
+    and pip install python-snappy
+
+Homebrew package is "snappy", in Ubuntu you need libsnappy1 and libsnappy-dev
+
+"""
+import os
+import sys
+import argparse
+import avro.schema
+from avro.datafile import DataFileWriter
+from avro.io import DatumWriter
+
+from amara.thirdparty import json
+from dplaingestion.couch import Couch
+from dplaingestion.selector import getprop
+
+avro_schema = """
+{
+    "namespace": "ingestion1.avro",
+    "type": "record",
+    "name": "record",
```

Comment:
I'm thinking about the namespace as well. Is it worth thinking now about how these records are in DPLA MAPv3.1, and how we might want to prevent their accidental loading into a future search index that has a MAP v4 schema?